pkg/handler/repositories.go

```diff
@@ -193,6 +195,14 @@ func (rh *RepositoryHandler) bulkCreateRepositories(c echo.Context) error {
 		return ce.NewErrorResponseFromError("Error creating repository", errs...)
 	}
 
+	// Send notifications
+	mappedValues := []repositories.Repositories{}
+	for i := 0; i < len(responses); i++ {
+		mappedValues = append(mappedValues, notifications.MapRepositoryResponse(responses[i]))
+	}
+
+	notifications.SendNotification(orgID, notifications.RepositoryCreated, mappedValues)
```

Comment:
how would you feel about passing `[]api.RepositoryRepsonse` to `SendNotifaction` and calling the mapping function in there instead?

Reply:
eh nevermind

Reply:
hmm.. We could do that, but the problem I see is that depending on the action, we might not have that specific struct readily available causing us to make a further dao call.

The repositories.Repositories{} struct also only **requires** a few of the variables, meaning that even though other structs may not map perfectly, they should be able to satisfy the required items in the  Repositories{} struct without additional dao calls.

Here are the actions that we will be sending notifications on, and thier related/most accessible structs.
* repository create
    * RepositoryResponse
* repository introspected
    * RepositoryUpdate?
* repository updated
    *  RepositoryUpdate
* repository introspected failed
    *  Repository
* repository deleted
    * We only send the UUID, we should likely call to get the repo info to send in the notification response.